data/sources/aerials-2018.json

```diff
@@ -0,0 +1,16 @@
+{
+    "id": "aerials-2018",
+    "type": "raster",
+    "tiles": [
+      "https://tiles.arcgis.com/tiles/yG5s3afENB5iO9fj/arcgis/rest/services/NYC_Orthos_2018/MapServer/tile/{z}/{y}/{x}"
+    ],
+    "tileSize": 256,
+    "meta": {
+      "description": "NYC OTI GIS Aerial Photography Tile Layers (TMS)",
+      "url": [
+        "https://tiles.arcgis.com/tiles/"
+    ],
+      "updated_at": "n/a"
+    }
+  }
+  
```

Comment:
It probably doesn't affect functionality but let's delete these empty lines at the end of all source files that have them for the sake of consistency.